src/content/docs/en/reference/experimental-flags/raw-env-values.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Experimental raw environment variables values
+sidebar:
+  label: Raw environment variables values
```

Comment:
```suggestion
  label: Raw environment variables
```
For the label, this might be a little long? Would this still be meaningful enough?